### ISSUE TITLE
[QUESTION] Speaker Diarization

### ISSUE THREAD

**Issue by slavakurilyak** [gitmylo/audio-webui#82]
# Speaker Diarization
Any plans to integrate a speaker diarization library, like [pyannote-audio](pyannote-audio) by @pyannote?

**Comment by gitmylo** [comment]
Good idea! This doesn't seem like something that will take long to implement, however, outputs will be as files probably, because gradio audio elements does not support an unknown amount of outputs, and you cannot add elements in gradio after you've already launched your app. Sadly. So a file output will have to do, outputting one audio file per speaker, or one audio file per separated speaker piece. It will be an option.
